src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -934,6 +934,7 @@ private function addSerializerSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->scalarNode('default_version')->end()
```

Comment:
From https://github.com/symfony/symfony/pull/38542 `serializer.default_context` config option is avalable now.

Why  not use `serializer.default_context.version` instead?